scalalib/src/mill/scalalib/GenIdea.scala

```diff
@@ -298,6 +298,7 @@ object GenIdea {
       <component name="NewModuleRootManager">
         <output url={"file://$MODULE_DIR$/" + relify(outputPath) + "/dest/classes"} />
         <exclude-output />
+        <content url={"file://$MODULE_DIR$/" + relify(outputPath) + "/.."} />
```

Comment:
You can do `outputPath / up` here

Reply:
Why do you need `/...` from the `outputPath` though? The `outputPath` itself should already be the folder that wraps the output paths of the module's Targets